src/cryptojwt/jwe/fernet.py

```diff
@@ -0,0 +1,37 @@
+import base64
+import os
+from typing import Optional
+from typing import Union
+
+from cryptography.fernet import Fernet
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+
+from cryptojwt import as_unicode
+from cryptojwt.jwe import Encrypter
+from cryptojwt.utils import as_bytes
+
+
+class FernetEncrypter(Encrypter):
+    def __init__(self, password: str, salt: Optional[bytes] = ""):
```

Comment:
`= None`  if optional?

Reply:
Not sure why None would be better than "" or b"" for that matter ?

Reply:
If you say `Optional`, it makes sense to make the default  `None` to indicate that no value (not even a zero length string) as provided. Mostly a question of style I guess.

Reply:
I guess :-)